tests/testUserAccount.py

```diff
@@ -0,0 +1,207 @@
+import json
+import unittest
+from psycopg2 import connect
+
+from application import create_app, db as database
+from application.models import dbname, user, host, password
+
+
+class SignTests(unittest.TestCase):
+
+    def setUp(self):
+        """Prepare testing environment."""
+
+        self.app = create_app('testing')
+        self.app = self.app.test_client()
+
+        self.user_data = {
+            "email": "meshmbuvi@gmail.com",
+            "username": "musyoka",
+            "driver": True,
+            "password": "mbuvi1",
+            "phone": "0719800509",
+            "confirm password": "mbuvi1"
+        }
+        self.passwords = {
+            "email": "meshmbuvi@gmail.com",
+            "username": "mbuvi1",
+            "driver": False,
```

Comment:
The variable namings between line 25 - 56 doesn't sound right.
Like the `self.passwords` variable I thought I will see passwords only to realize it means there's some change in the password but the dictionary maintains user data as its content.

Reply:
Thanks for the correction.Its updated now